api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -128,13 +162,49 @@ type VolumeReplicationGroupSpec struct {
 
 	Async VRGAsyncSpec `json:"async,omitempty"`
 	Sync  VRGSyncSpec  `json:"sync,omitempty"`
+
+	// volsync defines the configuration when using VolSync plugin for replication.
+	//+optional
+	VolSync VolSyncSpec `json:"volSync,omitempty"`
+
+	// PrepareForFinalSync when set, it tells VRG to prepare for the final sync from source to destination
+	// cluster. Final sync is needed for relocation only, and for VolSync only
+	//+optional
+	PrepareForFinalSync bool `json:"prepareForFinalSync,omitempty"`
+
+	// runFinalSync used to indicate whether final sync is needed. Final sync is needed for
+	// relocation only, and for VolSync only
+	//+optional
+	RunFinalSync bool `json:"runFinalSync,omitempty"`
```

Comment:
I understand that volsync is only used for Async/Regional DR, therefore it seems its inputs should be stored in the `VRGAsyncSpec`

Reply:
Agreed.  I created an issue [here](https://github.com/RamenDR/ramen/issues/459)